test/telephone_number/files/telephone_number_data_file_override.xml

```diff
@@ -0,0 +1,250 @@
+<!-- Copyright (C) 2009 The Libphonenumber Authors
```

Comment:
In terms of the file, we do need this for tests. I mean, I can create a contrived example file but I'm not sure this has any super specific MOBI business-domain logic. Thoughts?